src/domains/load-test/01_grafana.tf

```diff
@@ -0,0 +1,28 @@
+resource "azurerm_resource_group" "load_test" {
+
+  name     = "${var.prefix}-${var.env_short}-load-test-rg"
+  location = var.location
+
+}
+
+module "grafana_managed" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//grafana?ref=feature/grafana"
+
+  name = "${var.env_short}-grafana"
+
+  resource_group_name = azurerm_resource_group.load_test.name
+
+  tags = var.tags
+
+}
+
+resource "azurerm_dns_cname_record" "grafana" {
+
+  name                = "grafana.${var.dns_zone_prefix}.${var.external_domain}"
```

Comment:
This is only the record dns, not the entire fqdn

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/dns_cname_record

Reply:
```suggestion
name                = "grafana"
```